### PR TITLE
Clarifying import/export process in the Settings menu

### PR DESCRIPTION
**Pull request by biohzrddd**
Importing a file wasn't obvious after already starting a test budget. This PR adds Import to the Settings > File menu, includes a brief explanation of the process, and copies the "Sign out" button. Explanation added for Export as well.

### CODE REVIEW

File: packages/desktop-client/src/components/Settings.js [hunk 1]
```diff
@@ -428,9 +428,28 @@ function FileSettings({
...
 
-      <View style={{ marginTop: 30, alignItems: 'flex-start' }}>
+      <View style={{ marginTop: 30, maxWidth: 600, alignItems: 'flex-start' }}>
+        <Title name="Import" />
+        <Text style={{ lineHeight: '1.4em' }}>
+          Importing creates a new budget file and must be done after you are
+          signed out from the server. Once signed out, there is an option to
+          import a file.
+        </Text>
+        <Button style={{ marginTop: 10 }} onClick={() => signOut()}>
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/Settings.js:438]: **🥜 nitpick:** ‏
```suggestion
        <Button style={{ marginTop: 10 }} onClick={signOut}>
```

**Review by MatissJanis — APPROVED** [review]
Hey

Just a small nitpick from me. Overall: looks good! I'll leave the PR open for a bit longer though to see if anyone else has some feedback.

Thanks for your contribution!

**Comment by biohzrddd** [comment]
> 

Thanks for the review. My only thought would be merging my "import" addition with @j-f1 changes from https://github.com/actualbudget/actual/pull/267.

**Comment by MatissJanis** [comment]
> Thanks for the review. My only thought would be merging my "import" addition with @j-f1 changes from https://github.com/actualbudget/actual/pull/267.

Thanks for the call-out. Yes, I'll try to review @j-f1 PR in the coming days and merge his change first as it's quite a bit bigger. And after that we can work on adding the `Import` block in this PR.

**Comment by tjfinlinson** [comment]
I took a look this just a few minutes ago, and I'd recommend changing the button text to say close file instead of sign out. That matches what you do when you click at the top by the budget name in that small drop down menu. Or at the very least it seems like it would be a good idea to match those 2 things up so it's consistent.

**Comment by biohzrddd** [comment]
> 

I was going off of the top right Server > Sign out.

**Comment by tjfinlinson** [comment]
Gotcha. That makes sense. I guess there are really 2 different things you can do. Sign out to require you to enter the server password again, or you can just close the file and either open a new budget file, or import a backup to create  a new budget file. Do you think those 2 things should be kept separate and then documented?

**Comment by biohzrddd** [comment]
> 

Closing the file is the right option, thanks for pointing me in that direction. See my latest commit that changes it. https://github.com/actualbudget/actual/pull/405/commits/cdd9e58cf470925617d2bacd616ae8074430e04f

**Comment by MatissJanis** [comment]
:wave: we just merged #267 . Once you fix the merge conflicts we can see if we can merge this one too.

**Comment by rich-howell** [comment]
Hey @biohzrddd just checking in to see if this PR is something you are still willing to work on? Actual is getting some traction now and would love this to get merged.

**Comment by MatissJanis** [comment]
👋 I'll close this PR as it's been stale for quite a while. Please feel free to re-open if you resume working here.

**Pull request closed without merging**